genestack_client/data_importer.py

```diff
@@ -583,6 +586,7 @@ def create_dictionary(self, parent=None, name=None, url=None, metainfo=None):
         metainfo = metainfo or BioMetainfo()
         name and metainfo.add_string(BioMetainfo.NAME, name)
         url and metainfo.add_external_link(BioMetainfo.DATA_LINK, url)
+        term_type and metainfo.add_string('genestack.dictionary:termType', term_type)
```

Comment:
I strongly dislike using `and` in this way instead of `if` (it should be used in assignments only), but since it's all over the `DataImporter`, I'll just grumble a bit.